### PR TITLE
Data : KrishChopra22 ✅

### PR DESCRIPTION
**Pull request by KrishChopra22**
One more contribution to EddieHub Community! 🥳
Created linkfree profile - the open source project ✔

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2780"></a>

### CODE REVIEW

**Review by dan-mba — CHANGES_REQUESTED** [review]
Remove the commas on lines 11, 53, & 79

**Review by dan-mba — APPROVED** [review]
Looks good to me

**Comment by KrishChopra22** [comment]
Thanks for reviewing 😊

File: data/KrishChopra22.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
+{
+  "name": "Krish Chopra",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Open Source Enthusiast, Mobile App Developer",
+  "avatar": "https://user-images.githubusercontent.com/77012237/211071479-62ce9139-97e3-4314-844e-605575f69fb8.png",
```
Comment (Panquesito7) [inline data/KrishChopra22.json:6]: It's best to use this link as it's shorter and updates automatically.
```suggestion
  "avatar": "https://github.com/KrishChopra22.png",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Comment by KrishChopra22** [comment]
I updated my github profile pic recently, but this didn't reflect to my LinkFree.
How can it be updated dynamically?

**Comment by dan-mba** [comment]
> I updated my github profile pic recently, but this didn't reflect to my LinkFree. How can it be updated dynamically?

LinkFree caches avatar images for 24 hours. Check back in a day & if it still hasn't changed, open an issue or ask for help on EddieHub discord.

**Pull request merged**